tests/lib/rules/avoid-leaking-callbacks-in-ember-objects.js

```diff
@@ -0,0 +1,157 @@
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/avoid-leaking-callbacks-in-ember-objects');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+
+const eslintTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  }
+});
+eslintTester.run('avoid-leaking-callbacks-in-ember-objects', rule, {
+  valid: [
+    {
+      code: `
+export default Ember.Component.extend({
```

Comment:
This test case seems to duplicate the other valid test case. Let's keep the `Component.extend` but not the older syntax `Ember.Component.extend` one.

Reply:
Duplicate case removed.